package.json

```diff
@@ -37,13 +37,14 @@
   "homepage": "https://github.com/mapbox/pbf",
   "dependencies": {
     "ieee754": "^1.1.6",
-    "resolve-protobuf-schema": "^2.0.0"
+    "resolve-protobuf-schema": "~2.0.0"
```

Comment:
What's the reason for the range change?

Reply:
Explained in the commit comment here: https://github.com/mapbox/pbf/pull/91/commits/8c01f6b8a4287dc04a6b996992de546194c05a4f